### ISSUE TITLE
Why is BMS115 recommended?

### ISSUE THREAD

**Issue by robhu** [BotoX/xiaomi-m365-firmware-patcher#4]
You state "Update your BMS firmware to the latest version: BMS115.bin".

Why should BMS 115 be used and not BMS 107? I have been given the impression that 107 allows more power delivery.

Could you please explain the difference between them, why 115 is recommended, and why 107 (presumably) should not be used?

**Comment by BotoX** [comment]
I have no clue why people keep on suggesting BMS107, it does not deliver more power or do anything magic.

However I've heard multiple times that BMS107 calculates the remaining battery wrong however.

I'm sure Xiaomi wouldn't update the battery management system software without having a reason to do so (bugs, issues, safety, etc.)
So yeah I'm just recommending people to update their stuff instead of using outdated firmware which doesn't improve anything and could have bugs...

**Comment by kamesen99** [comment]
Hi. I downloaded BMS115 from the website and used the PATCHED APK to flash it. I got to 3% progress and the unit is now locked and can't find it with Bluetooth. Any suggestions?

**Comment by Vagos696** [comment]
I would like to provide my feedback since I was trying yesterday to create my custom firmware.
For my case BMS115 didn't operate properly and the max speed was around 16km/h. 
My main goal was to unlock the max speed and to change the kick start speed. Even when I was trying to send predifined firms like 200cfw the scooter was underforming significantly.
As soon as I reverted to BMS 107 everything were as expected. Even with predefined firms or by creating my own. 
I had to leave my experience since bms115 was not working for me. (I don't know the reason)

Also for the readers that they are trying to send firmware with the latest firm from xiaomi they have to downgrade first with downG_patched. Else the flash will not get through.
